stefaniaPopescu26/assignment2/Tree.java

```diff
@@ -0,0 +1,91 @@
+public class Tree<T> {
+    private Node<T> root;
+    private String inorder;
```

Comment:
This doesn't have to be a global variable. You only need it in the inorderTraversal method. Global variables should only be added if they hold a state that needs to be accessed at different times and from different methods (like root in this class). But the inorder can be locally computed in the inorderTraversal and returned as a return value there.